### PR TITLE
Rotl post

### PR DESCRIPTION
**Pull request by dwinter**
A post from me, @fmichonneau and @josephwb briefly promoting/celebrating the publication of a paper describing rotl

Adds a post in markdown and an image that forms part of that post, unfortunately I can't the versions of ruby, jekyll and associated plugins on my laptop to line up, so this is untested

### CODE REVIEW

**Comment by karthik** [comment]
@dwinter Post looks good to me and ready to merge. Thank you!
I will wait for @sckott and others to weigh in before we merge.

File: _posts/2016-07-18-rotl-paper-published.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      - name: David Winter
+        url: http://david-winter.info
+    tags:
+    - R
+    - rotl
+    - open-tree
+    - publication
+    ---
+
+We are excited to say a paper describing `rotl`, our package for the
```
Comment (sckott) [inline _posts/2016-07-18-rotl-paper-published.md]: maybe  s/say/announce

File: _posts/2016-07-18-rotl-paper-published.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+    ---
```
Comment (sckott) [inline _posts/2016-07-18-rotl-paper-published.md]: Left align the yaml header

File: _posts/2016-07-18-rotl-paper-published.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+give R users the ability to search for phylogenetic information and
+import the results into their R sessions. The imported data can then be
+used with the growing ecosystem of packages for phylogenetic and
+comparative biology in R.
+
+A small example
+---------------
+
+Using `rotl` to get a tree for a set of names is usually two step
+process. You need to start by matching the set of names you have to
```
Comment (sckott) [inline _posts/2016-07-18-rotl-paper-published.md]: `usually a two`

File: _posts/2016-07-18-rotl-paper-published.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+and used to [reproduce a comparative
+analysis](https://cran.rstudio.com/web/packages/rotl/vignettes/meta-analysis.html).
+[Another
+vignette](https://cran.rstudio.com/web/packages/rotl/vignettes/how-to-use-rotl.html)
+provides and introduction to the package and includes an FAQ.
+
+Thanks
+------
+
+We'd like to thank the Open Tree project for providing such an amazing
```
Comment (sckott) [inline _posts/2016-07-18-rotl-paper-published.md]: reword this sentence 

> We'd like to thank the Open Tree project for providing such an amazing resource and produce and powerful and well-documented API

**Comment by sckott** [comment]
@dwinter made a few comments, otherwise looks good

**Comment by dwinter** [comment]
Thanks @sckott , finally finished travelling and had a moment to update this, (updated it to appear tomorrow).

**Comment by sckott** [comment]
looks good, will post tomorrow morning

**Pull request merged**